lib/motions/general-motions.coffee

```diff
@@ -471,14 +471,20 @@ class ScrollKeepingCursor extends Motion
 
   execute: (count) ->
     # TODO: remove this conditional once after Atom v1.1.0 is released.
-    if @editor.setFirstVisibleScreenRow?
-      newTopRow = @getNewFirstVisibleScreenRow(count)
-      super(count)
-      @editor.setFirstVisibleScreenRow(newTopRow)
-    else
-      scrollTop = @getNewScrollTop(count)
-      super(count)
-      @editorElement.setScrollTop(scrollTop)
+    # TODO: Atom v1.1.0 was released.
+    # But setFirstVisibleScreenRow() use getLineCount() not
+    # getScreenLineCount(). Once after it use getScreenLineCount(),
+    # [1. by setFirstVisibleScreenRow] can be changed to [2. by scrollTop]
```

Comment:
@b6pzeusbc54tvhw5jgpyw8pwz2x6gs Are you interested in submitting a PR to atom/atom fixing this? I can publish a hotfix release of Atom so that we don't need to wait for 1.7.0 to come out.